src/ripple/rpc/handlers/Submit.cpp

```diff
@@ -246,13 +246,11 @@ doSubmitGrpc(
         }
     }
 
-    std::string reason;
-    auto tpTrans = std::make_shared<Transaction>(stpTrans, reason, context.app);
+    auto tpTrans = std::make_shared<Transaction>(stpTrans);
     if (tpTrans->getStatus() != NEW)
     {
         grpc::Status errorStatus{
-            grpc::StatusCode::INVALID_ARGUMENT,
-            "invalid transaction: " + reason};
+            grpc::StatusCode::INVALID_ARGUMENT, "invalid transaction"};
         return {result, errorStatus};
     }
```

Comment:
This block should be impossible to reach now. It can probably be safely replaced with `assert(tpTrans->getStatus() == NEW);`